File: arcgisapp/src/main/java/com/example/arcgisapp/MainActivity.java
package com.example.arcgisapp;

import android.view.View;

import com.example.arcgisapp.arcGIS.ArcGISActivity;
import com.example.arcgisapp.base.BaseToolbarActivity;

public class MainActivity extends BaseToolbarActivity {

    @Override
    protected int getContentViewLayoutID() {
        return R.layout.activity_main;
    }

    @Override
    protected void initViewsAndEvents() {
        mTvTitle.setText("首页");
    }

    public void goArcGISActivity(View view) {
        readyGo(ArcGISActivity.class);
    }
}

File: mykotlin/src/main/java/wjf/com/mykotlin/MainActivity.kt
package wjf.com.mykotlin

import wjf.com.mykotlin.base.BaseTitlebarActivity

class MainActivity : BaseTitlebarActivity() {

    override val contentViewLayoutID: Int
        get() = R.layout.activity_main

    override fun initViewAndData() {
        //隐藏toolbar返回按钮
        supportActionBar!!.setDisplayHomeAsUpEnabled(false)
        mTvTitle.setText("Kotlin 首页")
    }
}
